src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -0,0 +1,94 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    TouchableOpacity, Text,
+} from 'react-native';
+import CONST from '../../CONST';
+import HeaderWithCloseButton from '../HeaderWithCloseButton';
+import Modal from '../Modal';
+import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
+
+/**
+ * Modal render prop component that exposes modal launching triggers that can be used
+ * to display a full size image or PDF modally with optional confirmation button.
+ */
+
+const propTypes = {
+    // Optional callback to fire when we want to trigger the update.
+    onConfirm: PropTypes.func,
```

Comment:
I think we should rename this from `onConfirm` to `onSubmit`, to match the underlying modal prop.